instrumentation/aws_lambda/README.md

```diff
@@ -0,0 +1,60 @@
+# OpenTelemetry AWS-Lambda Instrumentation
+
+The OpenTelemetry `aws-lambda` gem is a community maintained instrumentation for [AWS Lambda functions](https://docs.aws.amazon.com/lambda/latest/dg/ruby-handler.html).
+
+## How do I get started?
+
+Install the gem using:
+
+```
+gem install opentelemetry-instrumentation-aws_lambda
+```
+
+Or, if you use [bundler][bundler-home], include `opentelemetry-instrumentation-aws_lambda` in your `Gemfile`.
+
+## Usage
+
+From the Lambda Layer side, create the wrapper. More information can be found at https://github.com/open-telemetry/opentelemetry-lambda/tree/main/ruby
+```ruby
```

Comment:
I was kind of surprised when I took a look at the opentelemetry lambda's [ruby/layer/src](https://github.com/open-telemetry/opentelemetry-lambda/tree/main/ruby/src/layer) directory. I didn't see a `wrapper.rb` file. Is this something that users need to make? 
Is there more that needs to be done to the layer repo before both pieces are ready?

Reply:
Yes, I will work on creating the wrapper.rb in that directory once this pr is ready (or published).
Since wrapper.rb needs the lambda instrumentation to work, so I thought to wait for this PR.
For example, [python wrapper.rb](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel_wrapper.py#L40) needs the instrumentation as well.

Reply:
Got it, that makes sense. Thank you!